src/pages/instances/InstancePagination.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Icon, Input, Select } from "@canonical/react-components";
+import React, { FC } from "react";
+
+interface Props {
+  pageSize: number;
+  setPageSize: (val: number) => void;
+  currentPage: number;
+  setCurrentPage: (val: number) => void;
+  visibleCount: number;
+  totalCount: number;
+  totalPages: number;
+}
+
+const InstancePagination: FC<Props> = ({
+  pageSize,
+  setPageSize,
+  currentPage,
+  setCurrentPage,
+  visibleCount,
+  totalCount,
+  totalPages,
+}) => {
+  return (
+    <div className="pagination-footer">
+      <div className="description">
+        Showing {visibleCount} out of {totalCount} instances
+      </div>
+      <Button
+        className="back"
+        appearance="base"
+        hasIcon
+        onClick={() => {
+          if (currentPage > 1) {
+            setCurrentPage(currentPage - 1);
+          }
+        }}
+      >
+        <Icon name="chevron-down" />
+      </Button>
+      <Input
+        className="u-no-margin--bottom"
+        onChange={(e) => {
+          const newPage = Math.min(
+            totalPages,
+            Math.max(1, parseInt(e.target.value))
+          );
+
+          setCurrentPage(newPage);
+        }}
+        value={currentPage}
+        type="number"
+      />{" "}
+      of&nbsp;{totalPages}
+      <Button
+        className="next"
+        appearance="base"
+        hasIcon
+        onClick={() => {
+          if (currentPage < totalPages) {
+            setCurrentPage(currentPage + 1);
+          }
+        }}
+      >
+        <Icon name="chevron-down" />
+      </Button>
+      <Select
+        options={[
+          {
+            label: "50/page",
+            value: 50,
+          },
+          {
+            label: "100/page",
+            value: 100,
+          },
+          {
+            label: "200/page",
+            value: 200,
+          },
+        ]}
```

Comment:
Can we extract these options to a `const`, like we already do for other options in the app, so that it becomes easier to edit them or add new ones in the future?